lib/players/human.rb

```diff
@@ -0,0 +1,11 @@
+class Human < Player
+  attr_accessor :human_move, :board
+
+  def move(board)
+    puts "Where do you want to move, #{self.token}? 1-9:"
+    @human_move = gets
+    human_move
```

Comment:
This last line isn't necessary; `human_move` will already be returned by the assignment operation in the previous line.